dspace-api/src/main/java/org/dspace/browse/SolrBrowseDAO.java

```diff
@@ -227,21 +223,14 @@ private DiscoverResult getSolrResponse() throws BrowseException {
     }
 
     private void addStatusFilter(DiscoverQuery query) {
-        if (itemsWithdrawn) {
-            query.addFilterQueries("withdrawn:true");
-        } else if (!itemsDiscoverable) {
-            query.addFilterQueries("discoverable:false");
-            // TODO
-
-            try {
-                if (!authorizeService.isAdmin(context)
-                    && (authorizeService.isCommunityAdmin(context)
-                    || authorizeService.isCollectionAdmin(context))) {
-                    query.addFilterQueries(searcher.createLocationQueryForAdministrableItems(context));
-                }
-            } catch (SQLException ex) {
-                log.error(ex);
+        try {
+            if (!authorizeService.isAdmin(context)
+                && (authorizeService.isCommunityAdmin(context)
+                || authorizeService.isCollectionAdmin(context))) {
+                query.addFilterQueries(searcher.createLocationQueryForAdministrableItems(context));
             }
+        } catch (SQLException ex) {
+            log.error(ex);
```

Comment:
While I realize this code isn't new in this PR, can we take this opportunity to improve the log statement here?  I think this should say something like:
`log.error("Error looking up authorization rights of current user", ex);`